README.md

```diff
@@ -154,10 +154,11 @@ It will :
 1. Install all dependencies.
 2. Install pre-commit hooks.
 
-Next, configure the environment variables in [dev.env](dev.env) to match your Athena development environment.
+Next, copy your `dev.env` file by copying the `dev.env.example` file and configure the environment variables
```

Comment:
Makefile snippet to make this more or less automatic:

```
dev.env:
	@echo "# AUTOGENERATED from dev.env.example please adjust" > $@
	@echo "" >> $@
	@cat dev.env.example >> $@
	@echo "PLEASE ADJUST .dev.env to your liking and rerun!"
	@false


.PHONY: test
test: dev.env  
	pytest ...
```

the result is that the first `make test` will output the message and fail, but the second will work...

Reply:
If we work locally is fine to have make test failing on the first attempt, but definitely this approach cannot be used in the ci. As I generally try to use the same make command locally and on ci, I suggest to change or improve this approach

Reply:
If we have a CI config, I would put that specific config into `.github/test.env` and then copy that to '.env' during the CI setup in the github action: -> the dependency is there and so the test would work... 

You would anyway need to do that, as you would need to run make twice otherwise: on the first run, the env file isn't there and so can't be imported at the top of the makefile and so the tests wouldn't have access to the variables in there.